src/main/java/com/vmware/vipclient/i18n/messages/api/opt/ComponentOpt.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vipclient.i18n.messages.api.opt;
+
+import java.util.List;
+
+public interface ComponentOpt {
```

Comment:
on Singleton service side, componentlist API is put under product, so is it better rename the class name as 'ProductOpt'?

Reply:
I don't mind. Anyway, you used ProductOpt in your PR so you can merge it then.